community/alacerda6br.md

```diff
@@ -0,0 +1,11 @@
+# Alex Lacerda
+
+Sou autodidata e sempre gostei de tecnologia.
+
+Alguns conhecimentos que tenho no mundo da tecnologia:
+HTML
+CSS
```

Comment:
![CSS3](https://img.shields.io/badge/CSS3-000?style=for-the-badge&logo=css3&logoColor=264CE4)